src/app/components/core/developer/additional-services/list-additional-services.component.ts

```diff
@@ -149,34 +197,22 @@ export class ListAdditionalServicesComponent implements OnInit, OnDestroy {
         });
   }
 
-  public getAllServices() {
-   this.servicesApiService.getAllServices().
-      subscribe((data: Service) => {
-        this.servicesRegistry = data['services'].filter((s) => this.expectedServices.some(es => es.type == s.type));
-        const addedServices = this.servicesSchedules.filter(sch => this.servicesRegistry.some(({name}) => sch.name === name));      
-        // If we get expected services in the response of /service API then no need to make other (/installed, /available) API calls
-        if (addedServices.length === this.expectedServices.length) {
-          this.availableServicePkgs = [];
-          let serviceTypes = [];
-          this.expectedServices.forEach(function (s) {
-            serviceTypes.push(s["process"]);
-          });
-          this.getInstalledServices(serviceTypes);
-          this.hideLoadingText();
-          this.ngProgress.done();
-        } else {
-          this.showServices();
-        }
-      },
-        (error) => {
-          /** request done */
-          this.ngProgress.done();
-          if (error.status === 0) {
-            console.log('service down ', error);
-          } else {
-            this.alertService.error(error.statusText);
-          }
-        });
+  public checkServices() {
```

Comment:
What is checkServices?

Reply:
If schedule of all services are not available then need to check expected external services in /service API